File: 251001/Brutskaya/disCo/publisher/src/main/java/com/example/publisher/api/dto/responce/IssueResponseTo.java
package com.example.publisher.api.dto.responce;

import lombok.Data;

import java.time.Instant;

@Data
public class IssueResponseTo {

    private Long id;
    private Long userId;
    private String title;
    private String content;
    private Instant created;
    private Instant modified;
}

File: 251003/Pankratiev/rest/publisher/src/main/kotlin/bsuir/dc/publisher/dto/to/IssueResponseTo.kt
package bsuir.dc.publisher.dto.to

import java.time.LocalDateTime

data class IssueResponseTo(
    val id: Long = 0,
    val writerId: Long = 0,
    val title: String = "",
    val content: String = "",
    val created: LocalDateTime = LocalDateTime.now(),
    val modified: LocalDateTime = LocalDateTime.now()
)
